### PR TITLE
feat(es6): change to core-js

### PR DESCRIPTION
**Pull request by wesleycho**
- Change ES6 polyfill to core-js

### CODE REVIEW

File: src/polyfills.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import 'es6-shim';
+import 'core-js';
```
Comment (Foxandxss) [inline src/polyfills.ts]: you don't really need to import the whole core-js, you can load the shims only.

**Comment by wesleycho** [comment]
Made change as noted

File: src/polyfills.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import 'es6-shim';
+import 'core-js/shim';
```
Comment (Foxandxss) [inline src/polyfills.ts]: in reality it would be `core-js/es6/shim`

**Comment by wesleycho** [comment]
Changed again :)

**Pull request merged**